build/postgres/Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM postgres:11-alpine
 
-RUN apk add --update git make gcc musl-dev
+RUN apk add --update git make gcc clang llvm musl-dev
```

Comment:
clang and llvm added, because otherwise:
```
make: /usr/bin/clang: Command not found
make: *** [/usr/local/lib/postgresql/pgxs/src/makefiles/../../src/Makefile.global:1031: wal2json.bc] Error 127
```
and then
```
/bin/sh: /usr/lib/llvm9/bin/llvm-lto: not found
make: *** [/usr/local/lib/postgresql/pgxs/src/makefiles/pgxs.mk:239: install] Error 127
```